kickoff/jsonexportl10n.py

```diff
@@ -15,37 +15,66 @@
 from jsonexport import myjsonify
 
 
+BETA_AGGREGATION_KEYWORD = "beta"
+
+
 @app.route('/json/l10n/<releaseName>.json', methods=['GET'])
 def l10nExport(releaseName):
     # Export the l10n changeset for a product
     releaseTable = getReleaseTable(releaseName)
+
+    if releaseName.endswith(BETA_AGGREGATION_KEYWORD):
+        l10n_list = _aggregateBetaLocales(releaseTable, releaseName)
+    else:
+        l10n_list = _getLocalesByReleaseName(releaseTable, releaseName)
+
+    l10n_list["version"] = config.JSON_FORMAT_L10N_VERSION,
+    return myjsonify(l10n_list)
+
+
+def _aggregateBetaLocales(releaseTable, releaseName):
+    # We need to keep the first "b", so we can look up names like "Firefox-32.0b"
+    beta_keyword_length = len(BETA_AGGREGATION_KEYWORD) - 1
+    betas_substring = releaseName[:-beta_keyword_length]
```

Comment:
For beta, I am happy with this.
In the future, if we want to do the same for aurora (a2) & nightly (a1), this will break. 
But we don't have to manage that now.